pgmq-extension/test/sql/base.sql

```diff
@@ -67,11 +67,11 @@ SELECT * from {PGMQ_SCHEMA}.metrics_all();
 
 -- delete all the queues
 -- delete partitioned queues
-SELECT pgmq.drop_queue(queue, true)
+SELECT pgmq.drop_queue(queue)
```

Comment:
Let's keep the test for `pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)` and then add a test for `pgmq.drop_queue(queue_name TEXT)`.